WordPress/Classes/Stores/StatsInsightsStore.swift

```diff
@@ -191,9 +233,15 @@ private extension StatsInsightsStore {
         }
     }
 
+    func apiService(`for` site: Int) -> StatsServiceRemoteV2 {
+        let api = WordPressComRestApi(oAuthToken: SiteStatsInformation.sharedInstance.oauth2Token, userAgent: WPUserAgent.wordPress())
```

Comment:
Due to changes requested via `p77Llu-byF-p2`, can we please amend this to the following?
```
let api = WordPressComRestApi(oAuthToken: SiteStatsInformation.sharedInstance.oauth2Token, userAgent: WPUserAgent.wordPress(), localeKey: WordPressComRestApi.LocaleKeyV2)
```

Reply:
Please disregard the above. I see now that `StatsServiceRemoteV2` an endpoint with a version of `1.1`.